src/style.css

```diff
@@ -1,3 +1,68 @@
+:root {
+  --is-active-color: #00acdc;
+}
+
 body {
-  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.headerLogo {
+  vertical-align: bottom;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-self: flex-end;
+}
+
+.nav__item {
+  margin: 0 10px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  font-family: Roboto, sans-serif;
```

Comment:
you already have set it for body tag

Reply:
I see it now. Thanks a lot for finding another issue!